src/vim.js

```diff
@@ -5625,7 +5629,7 @@ export function initVim(CodeMirror) {
 
           // Record the streams position at the beginning of the loop for use
           // in error messages.
-          var count = stream.pos;
+          var count = stream. pos;
```

Comment:
looks good, but could you please remove this change.

Reply:
Ah, whoops, my bad!

Reply:
Should be fixed in latest push. Please let me know if there's anything else I missed :)